### ISSUE TITLE
Rewrite the adapter using real adapter

### ISSUE THREAD

**Issue by JenyaKirmizaTripTop** [mutualmobile/CardStackUI#28]
Thank you sir, for a good library, though i have some concerns about its performance. You are using strange approach for adapter.
For example in setAdapter() method:

```
        for (int i = 0; i < mAdapter.getCount(); i++) {
            mAdapter.addView(i);
        }
```

It's bad cause all views are always in RAM, so it slows the performance dramatically. What if i have about 30 custom views. It will be lagging. Consider creating honest adapter for this purposes.

**Comment by sailywebvillee** [comment]
Hello , 
Have you find any solution for this problem. Please I am facing same issue.

**Comment by JenyaKirmizaTripTop** [comment]
@sailywebvillee no, i didn't, we had different design, so i'm not using this lib. The only way is to write it by yourself or use some other solution
